btr-web/btr-main-app/cypress/e2e/pages/summary-table/summaryTable.cy.ts

```diff
@@ -0,0 +1,217 @@
+        .get('[data-cy="address-region-select"]').contains(testData.profile1.address.province[1])
+        .get('[data-cy="address-postal-code"]').should('have.value', testData.profile1.address.postalCode)
+        .get('[data-cy="countryOfCitizenshipRadioGroup"]').get('[type="radio"][value="citizen"]').should('be.checked')
+        .get('[name="taxNumber"]').should('have.value', testData.profile1.taxNumber)
+        .get('[data-cy="testTaxResidency"]').get('[type="radio"][value="true"]').should('be.checked')
+    })
+  })
+
+  it('test the CANCEL button in the Edit form', () => {
+    cy.fixture('individuals').then((testData) => {
+      cy.get('[data-cy=summary-table-name]').contains(testData.profile1.fullName.toUpperCase())
+
+      cy.get('[data-cy=edit-button]').first().click()
+
+      // When CANCEL button is clicked, the profile will not be changed
+      cy.get('#individual-person-full-name').clear().type('NEW NAME')
+      cy.get('[data-cy=new-si-cancel-btn]').click()
+      cy.get('[data-cy=summary-table-name]').contains(testData.profile1.fullName.toUpperCase())
+      cy.get('[data-cy=summary-table-name]').should('not.contain', 'NEW NAME')
+
+      // the edit form will be closed
+      cy.get('[data-cy=individualsSummaryTable]').should('not.contain', 'Edit an Individual')
+
+      // the edit buttons will be enabled
+      cy.get('[data-cy=edit-button]').eq(0).should('not.have.attr', 'disabled')
+      cy.get('[data-cy=edit-button]').eq(1).should('not.have.attr', 'disabled')
+
+      // the "Add an Individual" button will be enabled
+      cy.get('[data-cy=add-new-btn]').should('not.have.attr', 'disabled')
+    })
+  })
+
+  it('test the DONE button in the Edit form', () => {
+    cy.fixture('individuals').then((testData) => {
+      cy.get('[data-cy=summary-table-name]').contains(testData.profile1.fullName.toUpperCase())
+
+      cy.get('[data-cy=edit-button]').first().click()
+
+      cy.get('#individual-person-full-name').clear().type('NEW NAME')
+      cy.get('[data-cy=new-si-done-btn]').click()
+      cy.get('[data-cy=summary-table-name]').contains('NEW N')
+      cy.get('[data-cy=summary-table-name]').should('not.contain', testData.profile1.fullName.toUpperCase())
+
+      cy.get('[data-cy=individualsSummaryTable]').should('not.contain', 'Edit an Individual')
+      cy.get('[data-cy=edit-button]').eq(0).should('not.have.attr', 'disabled')
+      cy.get('[data-cy=edit-button]').eq(1).should('not.have.attr', 'disabled')
+      cy.get('[data-cy=add-new-btn]').should('not.have.attr', 'disabled')
+    })
+  })
+
+  it('test the REMOVE button in the Summary Table', () => {
+    for (let i = 0; i < 2; i++) {
+      cy.get('[data-cy=popover-button]').first().click()
+      cy.get('[data-cy=remove-button]').click()
+    }
+
+    cy.get('[data-cy="individualsSummaryTable"]').get('td')
+      .should('contain.text', 'No significant individuals added yet')
```

Comment:
this will only be true if the business had 2 existing individuals (which is why it fails in the CI). This should dynamically calculate how many existing individuals are there and verify there is at least 1. FUTURE: we should have a specific test business for these tests that doesn't get interfered with